samples/servlet-filters-gzip/src/gzip/service.clj

```diff
@@ -61,6 +61,11 @@
               ::http/container-options {:context-configurator (fn [c]
                                                                 (let [gzip-handler (GzipHandler.)]
                                                                   (.setGzipHandler c gzip-handler)
+                                                                  ;; by default, "GzipHandler" only operates on "GET" 's.
+                                                                  ;; It is possible to add other methods via:
+                                                                  ;; (.addIncludedMethods gzip-handler (into-array ["GET" "POST"]))
```

Comment:
Can we remove `"GET"`? Including it here is unnecessary although it does no harm to do so.

Reply:
Unnecessary, but it's nice to see it explicitly listed.

Reply:
Ok.